aopy/data/base.py

```diff
@@ -142,6 +142,32 @@ def load_preproc_exp_data(preproc_dir, subject, te_id, date, verbose=True, cache
         warnings.warn(f"Preprocessing errors found in {filename}:\n{metadata['preproc_errors']}")
     return data, metadata
 
+def load_preproc_emg_data(preproc_dir, subject, te_id, date, verbose=True, cached=True):
+    '''
+    Loads emg data from a preprocessed file.
+
+    Args:
+        preproc_dir (str): base directory where the files live
+        subject (str): Subject name
+        te_id (int): Block number of Task entry object 
+        date (str): Date of recording
+        verbose (bool, optional): check for preprocessing errors and print them (default True)
+        cached (bool, optional): whether to allow loading cached version of data (default True)
+
+    Returns:
+        dict: Dictionary of exp data
+        dict: Dictionary of exp metadata
+    '''
+    filename = get_preprocessed_filename(subject, te_id, date, 'emg')
+    preproc_dir = os.path.join(preproc_dir, subject)
+    data = load_hdf_group(preproc_dir, filename, 'emg_data', cached=cached)
+    #metadata = load_hdf_group(preproc_dir, filename, 'exp_metadata', cached=cached)
```

Comment:
we definitely want to create emg_metadata, it should at a minimum include the sampling rate of the data, but possibly also some information about which arrays were recorded. idk how we can get that info, we might need to add some metadata settings to bmi3d when you turn on quatt_sync and append them to the emg file after it's done recording